templates/containers/what-are-containers.html

```diff
@@ -0,0 +1,243 @@
+          loading="auto"
+          ) | safe
+        }}
+      </div>
+    </div>
+</section>
+
+<section class="p-strip">
+  <div class="u-fixed-width">
+    <h2>Container virtualization</h2>
+    <p>Containers are instantiated from container images, a layered packaging system, to distribute applications along with their configuration and runtime dependencies.</p>
+    <p>From development to production, containers power modern cloud applications as they allow developers to move fast between platforms and versions.</p>
+    <p>Inside Google alone, at least <a href="https://cloud.google.com/containers/" class="p-link--external">two billion containers are generated each week</a> to manage its vast operations. There are many tools to create, deploy and manage containers, including LXD, Docker and Kubernetes.</p>
+  </div>
+</section>
+
+<section class="p-strip--light">
+  <div class="u-fixed-width">
+    <h2>What is Docker?</h2>
+    <p><a href="https://www.docker.com/" class="p-link--external">Docker</a> is an open-source containerisation platform. It provides an easy way to build and deploy containers on the cloud or on-premises. Container technologies were not introduced by Docker, but they made the user experience of container technologies so easy for developers that Docker soon became shorthand for containers as well as the most popular container format. <a href="https://www.docker.com/" class="p-link--external">Docker Inc</a>, the company that supports the Docker project and community, also operates <a href="https://hub.docker.com/" class="p-link--external">Docker Hub</a>, a space for hosting container images.</p>
+  </div>
+</section>
+
+<section class="p-strip ">
+  <div class="u-fixed-width">
+    <h2>Kubernetes vs Docker: CRI-O and OCI</h2>
+    <p>With more and more applications deployed on containers, simple orchestration tools are not enough to support complex production environments. ‘<a href="/kubernetes/what-is-kubernetes#container-orchestration">Container orchestration</a>’ becomes a problem space of its own. The rise of <a href="/kubernetes">Kubernetes</a> is the industry's answer to the container orchestration problem, providing a way to manage containers at scale efficiently.</p>
+    <p>CRI (Container Runtime Interface) is a Kubernetes API to interface between Kubernetes orchestration features and container runtimes, such as containerd or CRI-O. The <a href="https://opencontainers.org/" class="p-link--external">Open Container Initiative</a> (OCI) defines a set of specifications to enable all these technologies to work together. Docker, runc, containerd, CRI-O are all OCI compatible.</p>
+  </div>
+</section>
+
+<section class="p-strip--light">
+  <div class="row">
+    <div class="col-6">
+      <p>Not clear on how Kubernetes vs Docker differ?</p>
+      <p><a href="/blog/kubernetes-versus-docker">Read the Kubernetes vs Docker blog&nbsp;&rsaquo;</a></p>
+    </div>
+    <div class="col-6 u-align--center u-vertically-center">
```

Comment:
I cannot find this image on the copy doc.

Reply:
We made some changes in a call, I will add them to the doc :)